src/plugins/search/search.js

```diff
@@ -96,29 +96,27 @@ export function search(query) {
 
   for (let i = 0; i < data.length; i++) {
     const post = data[i]
-    let isMatch = false
+    let matchesScore = 0
     let resultStr = ''
     const postTitle = post.title && post.title.trim()
     const postContent = post.body && post.body.trim()
     const postUrl = post.slug || ''
 
-    if (postTitle && postContent) {
-      keywords.forEach(keyword => {
+    if (postTitle) {
+      keywords.forEach( keyword => {
         // From https://github.com/sindresorhus/escape-string-regexp
-        const regEx = new RegExp(
+        let regEx = new RegExp(
```

Comment:
Why you changed it to `let`?

Reply:
No reason, I don't personally buy the modern const vs let usage so my usual let usage slipped up in this code. Changing.

Reply:
>> edit: done. Should I squash the commit?

Yes, please.

Reply:
done